### DISCUSSION TITLE
Laravel Voyager always load old data after update

### DISCUSSION THREAD

**Discussion by tyghaykal** [thedevdojo/voyager#5369] [Q&A]
## Version information

 - Laravel: v7.0.0
 - Voyager: v1.4
 - PHP: 7.4
 - Database: Mysqld  Ver 10.3.25-MariaDB-0+deb10u1 for debian-linux-gnu on x86_64
## Description

I use Voyager to make CMS page for my website, but everytime I update a data using voyager and I open it again, it always load the old data before updating, but if I just refresh the page, the page will load the latest one. But it will fix if I update and open the edit page with console log opened. If you have this problem before, please tell me the solution. Thank you.

**Comment by MrCrayon** [comment]
Without seeing the code you are using to collect data it's impossible to help you in any way.
I can tell you that Voyager does not have any caching system for BREAD data so your problem must be somewhere else.
